### PR TITLE
mkblctrl - identify added

### PR DESCRIPTION
**Pull request by NosDE**
Identify mode added (MKBLCTRL_TEST) to veryfy ESC Adresses.
Bugfix in rcS (Arguments)

### CODE REVIEW

File: src/drivers/mkblctrl/mkblctrl_params.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * Parameters defined by the mkblctrl driver.
+ *
+ * @author Marco Bauer <marco@wtns.de>
+ */
+
+#include <nuttx/config.h>
+#include <systemlib/param/param.h>
+
+/**
+ * ID of the board this parameter set was calibrated on.
```
Comment (LorenzMeier) [inline src/drivers/mkblctrl/mkblctrl_params.c]: comment lies

File: src/drivers/mkblctrl/mkblctrl_params.c [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ *
+ * @group Sensor Calibration
```
Comment (LorenzMeier) [inline src/drivers/mkblctrl/mkblctrl_params.c]: comment lies

**Comment by LorenzMeier** [comment]
I suggest to rebase on master, if Travis still fails you would need to fix it, but I expect it to pass then.

**Comment by NosDE** [comment]
On my machine it compiles without any Error. I don't really know why Travis-CI build failed.

**Pull request closed without merging**